src/main/java/com/crowd/bean/ProjectResponse.java

```diff
@@ -201,4 +201,10 @@ public double getDonateSubtractApprovePercent() {
     public void setDonateSubtractApprovePercent(double donateSubtractApprovePercent) {
         this.donateSubtractApprovePercent = donateSubtractApprovePercent;
     }
+
+    public boolean isProjectInProgress(){
```

Comment:
`Project` is redundant in this method name as object represents project already.